sources/api/migration/migrator/tests/data/repository/targets/a-second-migration.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eo pipefail
+migration_name="a-second-migration"
```

Comment:
It seems the only difference between the scripts is this name, and I think it can be `${0##*/}`, and then they're the same script, maybe just symlinked?

Reply:
> It seems the only difference between the scripts is this name, and I think it can be `${0##*/}`, and then they're the same script, maybe just symlinked?

Makes sense, I'll add it to the TODO list for test cleanup and do this when I generate the TUF repo at runtime.

Reply:
Addressed by:

> webern force-pushed the webern:pr/signed-migrations branch from f481a2d to 4443441